.circleci/config.yml

```diff
@@ -167,6 +167,7 @@ commands:
           command: |
             mkdir $BIN_DIR
             mv target/<< parameters.target >>/release/cargo-shuttle<< parameters.suffix >> $BIN_DIR/cargo-shuttle<< parameters.suffix >>
+            mv target/<< parameters.target >>/release/shuttle<< parameters.suffix >> $BIN_DIR/shuttle<< parameters.suffix >>
```

Comment:
The corresponding change in install.sh will be in a separate PR and shall be merged after the artifacts of the next release are published.